var/spack/repos/builtin/packages/scotch/package.py

```diff
@@ -140,9 +140,17 @@ def cmake_args(self):
 
         if "+int64" in self.spec:
             args.append("-DINTSIZE=64")
+        elif self.is_64bit():
+            c_flags = []
+            c_flags.append("-DIDXSIZE64")
+            c_flags.append("-DINTSIZE32")
+            args.append(self.define("CMAKE_C_FLAGS", " ".join(c_flags)))
 
         return args
 
+    def is_64bit(self):
+        return "64" in str(self.pkg.spec.target.family)
```

Comment:
Duplicate of below?

Reply:
Yes, it is duplicate since I do not know how to factor that ; when the function is not in the class it is not found.
I thought you could help me for that. Thanks for reviewing.

Reply:
I couldn't tell from the diff that this was in a different "Builder". Sorry!